src/cli/commands/gsn.ts

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env node
+
+import commander from 'commander'
+
+commander
+  .version('0.0.1')
+  .command('deploy', 'deploy RelayHub and other GSN contracts instances')
+  .command('relayer-register', 'stake for a relayer and fund it')
+  .command('relayer-run', 'launch a relayer server')
+  .command('paymaster-fund', 'fund a paymaster contract so it can pay for relayed calls')
```

Comment:
why long name? what bad with `gsn fund --paymaster=` ?

Reply:
Nothing is wrong, we can rename parameters at any time